.github/workflows/publish.yml

```diff
@@ -21,3 +21,18 @@ jobs:
       run: |
         pip install twine
         twine upload --skip-existing --verbose dist/*
+  generate_authors:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Generate Authors
+      run: sh ./scripts/generate-authors.sh
+    - name: Commit report
+      run: |
+        git config --global user.name 'GitHub Action'
+        git config --global user.email 'action@github.com'
```

Comment:
Out of curiosity, won't it be included in a next release as an author?

Reply:
Yes! 

But I think It can be easily handled on the script or even with the `.mailmap` in case we use one.